lib/ansible/plugins/inventory/netbox.py

```diff
@@ -246,6 +248,22 @@ def extract_device_role(self, host):
         except Exception:
             return
 
+    def extract_config_context(self, host):
```

Comment:
Addition of my changes will be seen as yours which is plagiarism

Reply:
I would suggest you to remove the config-context part and keep your services feature here then it will be ok